app/views/components/_quick-escape-banner.html.erb

```diff
@@ -3,6 +3,12 @@
   aria_label ||= "Hide this page. Quick exit button, it opens BBC Weather in a new tab"
   data_attributes ||= {}
   data_attributes[:id] = "app-quick-escape-banner-link"
+
+  onboarding_title ||= "What the ‘hide this page’ button does"
+  onboarding_information ||= sanitize("
+    <p class=\"govuk-body\">Use this button to quickly hide this page. For example, to stop someone seeing you're using this service. The button opens BBC Weather in a new tab.</p>
+    <p class=\"govuk-body\">This tab will stay open and also go to BBC weather to hide the page you are using, but when you go back in your browser you will go to the start page of this service. Your answers will not be stored.</p>
+  ")
```

Comment:
Using sanitize isn't necessary here because you've defined all the HTML, I think you're using it as a way to get a `.html_safe` string which would be easier understood directly.

I'm not convinced this text belongs in the component as it's both dependent on information defined outside the component (BBC weather) and Smart Answer session specific ("but when you go back in your browser you will go to the start page of this service. Your answers will not be stored."). I'd suggest it be changed to just something neutral, if we need a default such as: `<p class=\"govuk-body\">Use this button to quickly hide this page. For example, to stop someone seeing you're using this service.</p>`. And then the app specific code is set in app specific views.

Reply:
Makes sense - I'll update.